test/Interop/Cxx/class/inheritance/Inputs/inherited-lookup.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+struct Base1 {
+  int method(void) const { return 1; }
+};
+
+struct IBase1 : Base1 {};
```

Comment:
Could we add a method at each layer and test that all of them are properly imported?

Reply:
I've added a method at each layer, although I do believe there is already quite a lot of coverage for that in the rest of the Swift test suite.

My intention here is to ensure coverage for a very specific scenario I encountered while studying the implementation of `ClangRecordMemberLookup`, at least for now. Later, I plan on expanding this test suite in subsequent patches to handle issues like overridden methods and ambiguous lookups (which you had foreshadowed in another comment). These patches will all be necessary to build back up to https://github.com/swiftlang/swift/pull/77726 in an incremental and robust way.